debian/changelog

```diff
@@ -1,3 +1,24 @@
+pg-auto-failover-enterprise (1.0.3) testing; urgency=low
+
+  * Fix Terminate conflicting queries at failover time (#194) 
```

Comment:
debian changelogs should not contain the changelog of the application, but instead the changelog to the packaging. I have made that mistake in the past as well :).

Hence the boring changlog here that reads something like `Official release of <package> <version>`